e2e/e2e_test.go

```diff
@@ -1,8 +1,15 @@
+/*
+This package exists to wrap our e2e provisioning and test framework so that it
+can be run via 'go test ./e2e'. See './framework/framework.go'
+*/
```

Comment:
pedantic: format package comments like,
```
// Package e2e wraps ...
package e2e
```
https://blog.golang.org/godoc-documenting-go-code

Reply:
Fixed in aa3b15d